.github/workflows/release-admin-tools.yml

```diff
@@ -0,0 +1,46 @@
+name: Release Admin Tools Image
+
+permissions: read-all
+
+on:
+  workflow_dispatch:
+    inputs:
+      commit:
+        description: "Repo Commit sha"
+        required: true
+      admintools_tag:
```

Comment:
any chances this to get computed automatically based on the rules we discussed?

Reply:
I'll do this in a separate PR, as discussed